roles/pid_generator/files/gen_pids.py

```diff
@@ -161,20 +161,20 @@ def setHandle (urnNumber, url, db_con, db_cur):
                 
 
 def xmlPrintHeader(outFile):
-    print >> outFile, '<?xml version="1.0" encoding="ASCII"?>'
-    print >> outFile, '<records xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="urn:nbn:se:uu:ub:epc-schema:rs-location-mapping http://urn.kb.se/resolve?urn=urn:nbn:se:uu:ub:epc-schema:rs-location-mapping&amp;godirectly">'
-    print >> outFile, ' <protocol-version>3.0</protocol-version>'
-    print >> outFile, ' <datestamp type="modified">'+dateStamp+'</datestamp>'
-    print >> outFile, ''
+    print ('<?xml version="1.0" encoding="ASCII"?>', file=outFile)
```

Comment:
It seems that this file isn't black-compliant. Black should be added to dev requirements as there's python stuff, and then run on this (and preferably on other python files too while we are at it)